specs/destination.go

```diff
@@ -71,3 +71,11 @@ func (d *Destination) Validate() error {
 	}
 	return nil
 }
+
+func (d Destination) String() string {
+	pathParts := strings.Split(d.Path, "/")
+	if d.Name == pathParts[1] {
```

Comment:
Do we validate `d.Path` contains at least one `/` beforehand? otherwise we'll get an index out of range panic here.

Reply:
Nice catch! We do, but not soon enough for some log prints, I added https://github.com/cloudquery/plugin-sdk/pull/609/commits/83d728c697195291cb84d2053c5ac7581ab9879c to also handle non GitHub registries